### PR TITLE
Replace all occurences of "en/latest" with "en/stable"

### PR DESCRIPTION
**Pull request by jklaise**
This is to ensure that our docs always point to the stable release
instead of latest to avoid potential discrepancy between someone reading
the latest docs but installing a stable version.

### CODE REVIEW

**Comment by jklaise** [comment]
Same comments apply to this PR as here: https://github.com/SeldonIO/alibi-detect/pull/479#issuecomment-1090274958

File: README.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ Alibi can be installed from:
 
 ### With conda 
 
-To install from [conda-forge](https://conda-forge.org/) it is recommended to use [mamba](https://mamba.readthedocs.io/en/latest/), 
+To install from [conda-forge](https://conda-forge.org/) it is recommended to use [mamba](https://mamba.readthedocs.io/en/stable/),
```
Comment (mauicv) [inline README.md:124]: is this intentional. It's a `mamba` link not `alibi`?

**Review by mauicv — APPROVED** [review]
There are a lot of places where we've changed other library links to the stable docs but I assume that's intentional.

LGTM

**Comment by jklaise** [comment]
Yup that was intentional both for consistency and because users will almost always run `stable` versions of everything.

**Pull request merged**